test/presubmit-tests-sdk.sh

```diff
@@ -21,6 +21,12 @@ python3 -m pip install mock  # TODO: Remove when added to requirements
 python3 -m pip install -r "$source_root/sdk/python/requirements.txt"
 python3 -m pip install --upgrade protobuf
 
+# Testing the component authoring library
+pushd "$(mktemp --directory)"
+cp "$source_root"/sdk/python/{components,components_tests} .
```

Comment:
```suggestion
cp "$source_root"/sdk/python/kfp/{components,components_tests} .
```

I think this is the reason the tests fails as well

``` 
cp: cannot stat '/home/prow/go/src/github.com/kubeflow/pipelines/sdk/python/components': No such file or directory
cp: cannot stat '/home/prow/go/src/github.com/kubeflow/pipelines/sdk/python/components_tests': No such file or directory
```

Reply:
Fixed. Thank you.